custom/opm/reports.py

```diff
@@ -955,14 +955,30 @@ def awc_data(self):
 
     @property
     def rows(self):
+
+        totals = [[None, None] for i in range(len(self.model.method_map))]
+        def add_to_totals(col, val, denom):
```

Comment:
I disagree.  This function exists to mutate `totals` - I find it clearer this way.